ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -1387,6 +1436,17 @@ FEDataManager::interpWeighted(const int f_data_idx,
                 for (unsigned int i = 0; i < n_vars; ++i)
                 {
                     F_dof_map_cache.dof_indices(elem, F_dof_indices[i], i);
+                    // TODO: This probably should be cached.
+                    if (ghosted_vec)
+                    {
+                        F_local_dof_indices[i].resize(F_dof_indices[i].size());
+                        std::transform(F_dof_indices[i].begin(),
+                                       F_dof_indices[i].end(),
+                                       F_local_dof_indices[i].begin(),
+                                       [&](unsigned int idx) -> unsigned int {
+                                           return F_petsc_vec->map_global_to_local_index(idx);
+                                       });
```

Comment:
This is, AFAIK, exactly what we need to do right now (until we get #513 finished)